docs/database_migration_test.md

```diff
@@ -0,0 +1,47 @@
+## Testing database migration
+
+### Creating a reference database
+
+Databases are encrypted, the key to decrypt is needed to setup the test.
+A special build property must be enabled to extract it. 
+
+Set `vector.debugPrivateData=true` in `gradle.properties`
```

Comment:
To avoid committing this change by mistake, it's better to tell that user can add the following lines in the local file `~/.gradle/gradle.properties`:

```
vector.debugPrivateData=true
```